libs/core/algorithms/include/hpx/parallel/algorithms/adjacent_difference.hpp

```diff
@@ -409,12 +409,12 @@ namespace hpx {
         }
 
         // clang-format off
-        template <typename ExPolicy, typename FwdIter1, typename FwdIter2,
-            HPX_CONCEPT_REQUIRES_(
+        template <typename ExPolicy, typename FwdIter1, typename FwdIter2>
+            requires (
                 hpx::is_execution_policy_v<ExPolicy> &&
                 hpx::traits::is_iterator_v<FwdIter1> &&
                 hpx::traits::is_iterator_v<FwdIter2>
-            )>
+            )
         // clang-format on
```

Comment:
We may want to move the cmake-format directive to wrap the requires clauses only:
```suggestion
        template <typename ExPolicy, typename FwdIter1, typename FwdIter2>
            // clang-format off
            requires (
                hpx::is_execution_policy_v<ExPolicy> &&
                hpx::traits::is_iterator_v<FwdIter1> &&
                hpx::traits::is_iterator_v<FwdIter2>
            )
            // clang-format on
```